### ISSUE TITLE
Ship non-threads and threads SDKs as a single SDK

### ISSUE THREAD

**Issue by kateinoigakukun** [swiftwasm/swift#5590]
SwiftPM does not allow multiple targets within a single Swift SDK for now. There is no reason for the restriction and it forces us to distribute two SDKs for wasm32-unknown-wasip1-threads and wasm32-unknown-wasi.

**Comment by ephemer** [comment]
@finagolfin seems to have implemented multiple targets within the [Android SDK](https://github.com/finagolfin/swift-android-sdk), I'm curious whether that approach is accessible here?

**Comment by finagolfin** [comment]
Yeah, who says it's not allowed? The [Musl static SDK bundle also supports multiple targets](https://www.swift.org/documentation/articles/static-linux-getting-started.html).

**Comment by kateinoigakukun** [comment]
The problem is even the format allows multiple targets, tools cannot uniquely select one of them when multiple SDK bundles containing multiple targets. See https://github.com/swiftlang/swift-package-manager/issues/7973

**Comment by finagolfin** [comment]
OK, sure, that's a known bug, but it will only affect you if you're using the same target triple for threads and non-threads? Otherwise, why would you need multiple wasm SDK bundles anyway?

**Comment by kateinoigakukun** [comment]
> it will only affect you if you're using the same target triple for threads and non-threads?

It could also be a problem if we are installing multiple versions of Wasm SDKs. We are frequently updating our SDK, so different branches in some code base can use different SDKs.
